BackendIntegrationTests/BackendIntegrationTests.swift

```diff
@@ -48,17 +48,17 @@ class BackendIntegrationTests: XCTestCase {
         configurePurchases()
         var completionCalled = false
         var receivedError: Error? = nil
-        var receivedOfferings: Offerings? = nil
```

Comment:
I still don't like this at all 😅 in a strongly typed language we never put the type in variable names. This is like doing `completionCalledBoolean`. The fact that this is a "Maybe" (like Haskell calls it) is already encoded in the type.

Reply:
yeah, I hear ya. I do find it useful for callbacks in particular, though, and as an ocasional reminder to consider nullability. It might still be the python coder in me that won't let go.

Reply:
The compiler will remind you :P